### PR TITLE
Update driver

### PR DESCRIPTION
**Pull request by daikk115**

### CODE REVIEW

File: etc/calplus/calplus-config-generator.conf [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[DEFAULT]
+wrap_width = 80
+output_file = etc/calplus/calplus.conf.sample
+namespace = calplus.conf
+namespace = oslo.log
```
Comment (hieulq) [inline etc/calplus/calplus-config-generator.conf]: no newline at end of file

File: etc/calplus/calplus.conf.sample [hunk 2]
```diff
@@ -0,0 +1,210 @@
+[DEFAULT]
```
Comment (hieulq) [inline etc/calplus/calplus.conf.sample]: put this sample file in the gitignore list

File: etc/calplus/calplus.conf.sample [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+
+
+[openstack]
+
+#
+# From calplus.conf
+#
+
+# List of available OpenStack Hosts (dict value)
+#hosts = openstack1:{'os_project_name': 'admin', 'os_image_api_version': '2', 'os_novaclient_version': '2.1', 'os_identity_api_version': '3', 'tenant_id': 'fake_tenant_id', 'os_auth_url': 'http://host1:35357/v3', 'os_username': 'admin', 'limit': {'rbac_policy': -1, 'subnetpool': -1, 'security_group': 10, 'router': 10, 'port': 50, 'security_group_rule': 100, 'subnet': 10, 'floatingip': 50, 'network': 10}, 'os_password': 'ADMIN_PASS', 'os_project_domain_name': 'default', 'os_user_domain_name': 'default'},openstack2:{'os_password': 'ADMIN_PASS', 'os_identity_api_version': '3', 'os_project_name': 'admin', 'limit': {'rbac_policy': -1, 'subnetpool': -1, 'security_group': 10, 'router': 10, 'port': 50, 'security_group_rule': 100, 'subnet': 10, 'floatingip': 50, 'network': 10}, 'os_user_domain_name': 'default', 'os_project_domain_name': 'default', 'os_auth_url': 'http://host2:35357/v3', 'os_username': 'admin', 'os_image_api_version': '2', 'tenant_id': 'fake_tenant_id'}
```
Comment (hieulq) [inline etc/calplus/calplus.conf.sample]: ok, seems like all the config details were put on single line

File: etc/calplus/calplus.conf.sample (hunk 2 above)
Comment (daikk115) [inline etc/calplus/calplus.conf.sample]: Actually, I think we should have it on our project.

For instance:
https://github.com/openstack/glance/tree/master/etc

File: etc/calplus/calplus.conf.sample (hunk 3 above)
Comment (daikk115) [inline etc/calplus/calplus.conf.sample]: I thought I will make it in another pull request like you said.

File: etc/calplus/calplus.conf.sample (hunk 2 above)
Comment (hieulq) [inline etc/calplus/calplus.conf.sample]: no, look carefully on why glance need to put sample config file there and take a look around nova, cinder... 
I prefer delete it on our project.

File: etc/calplus/calplus-config-generator.conf (hunk 1 above)
Comment (hieulq) [inline etc/calplus/calplus-config-generator.conf]: hm, so why github still complains about this error?

File: etc/calplus/calplus.conf.sample (hunk 2 above)
Comment (hieulq) [inline etc/calplus/calplus.conf.sample]: hence, push another commit for merging

**Pull request merged**